packages/flutter_tools/lib/src/compile.dart

```diff
@@ -62,6 +62,7 @@ Future<String> compile(
     bool linkPlatformKernelIn : false,
     bool aot : false,
     bool strongMode : false,
+    bool trackWidgetCreation: false,
```

Comment:
while you're here, can you fix the formatting on the previous lines? (extraneous spaces before the colons)

Reply:
done